applications/prompt-proto-service-lsstcomcam/values.yaml

```diff
@@ -62,7 +62,7 @@ prompt-proto-service:
     # -- A factor by which to multiply `baseSize` for refcat datasets.
     refcatsPerImage: 4
     # -- A factor by which to multiply `baseSize` for templates and other patch-based datasets.
-    patchesPerImage: 4
+    patchesPerImage: 6
```

Comment:
I don't know what's the optimal values, but I probably would start with those of LSSTComCamSim. Or where did you get these values that I probably missed?

Reply:
These values came from LATISS but I will update them to LSSTComCamSim's.